### DISCUSSION TITLE
Love it! and...

### DISCUSSION THREAD

**Discussion by KamiyaMinoru** [adamlui/ai-web-extensions#17] [Ideas]
Hi! I love this very much. And I want to ask if it will work with https://copilot.microsoft.com/ as well in the future?

**Comment by adamlui** [comment]
Hey thanks for the love @KamiyaMinoru! Which app do you want to work with it?

**Reply by KamiyaMinoru** [reply]
Lui, sorry it's about wide screen (copilot=new bing)

**Reply by adamlui** [reply]
Ok I will try eventually after I finish many changes to the Google/DuckDuck/Brave -GPT apps
